sample/halles_des_vosges/README.md

```diff
@@ -0,0 +1,23 @@
+# Halles des Vosges, Belfort, France
+
+This sample is a demonstration of multi-material building and slopped roof.
+
+The construction method is based on the building divided in sectors. These sectors are specified by their angles, and have no fixed amount of sides, in the *.mif* and *.mid* files. The slopes are specified in each *.ext* file. In this case 18 files are sufficient to describe all 28 sectors.
```

Comment:
I believe this paragraph should be rewritten along these lines:

1. describe what the building is (mainly glass walls with concrete bottom). Since we cannot quite use a real life picture of it, maybe it is a good idea to introduce the resulting model (figure 2/3) here to help.
1. explain that the building was split into 28 parts so that each require a single ext file.
1. explain that disjoint parts can actually share the same ext file (give an example).
2. explain that on the contrary maybe some parts had to be split apart because of inner material differences.
3. explain that dedicated parts were provided for the walls because it would otherwise have created a concrete floor/ceiling at midheight.

Do not forget to introduce figures in the main text. A figure is supposed to support text, hence the text should mention them.